src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -793,34 +794,34 @@ export class AppComponent implements OnInit, AfterViewChecked {
     /**
      *
      */
-    keyEvent(e: KeyboardEvent): void {
-        const self = this;
-        let eString = this.shortcuts.buildEventString(e);
-        this.shortcuts.getEvent(eString).then((result) => {
+    async keyEvent(e: any): Promise<void> {
+        if (e.target.classList.contains('slick-cell')) {
+            $('#messages').css('user-select', 'none');
```

Comment:
So you're saying that putting a user-select: contain on the results and messages container doesn't work? It's still HTML in the end and my understanding is that while you hook up to the events for selecting the grids properly that you could at least prevent the selection of the messages text by containing the select-all functionality.

Reply:
having user-select any other than none when selecting all from the grid will also select the messages unfortunately (and also the context menu if you right click before selecting all)

Reply:
That's odd. Wonder if it's because of the slickgrid stuff...anyways this is messy as heck and I really don't like it but I don't have a better solution at the moment so if you've tried some other things and can't get anything else to work I guess this is fine. I'd suggest running it by @anthonydresser  though since it involves the slickgrid stuff and he's a lot more familiar with that...

Reply:
I don't remember anything about this since i think this was added after I left the team. But select all in the grid should be able to entirely be handled by the grid, we shouldn't have to worry about the messages.